src/dsp_tools/commands/excel2json/new_lists/compliance_checks.py

```diff
@@ -236,23 +231,31 @@ def _check_for_missing_translations_one_sheet(sheet: ExcelSheet) -> MissingTrans
     languages = get_all_languages_for_columns(sheet.df.columns)
     all_cols = _compose_all_combinatoric_column_titles(col_endings, languages)
     problems = []
-    for column_group in all_cols.node_cols:
-        problems.extend(_check_for_missing_translations_one_column_level(column_group.columns, sheet.df))
-    problems.extend(_check_for_missing_translations_one_column_level(all_cols.list_cols.columns, sheet.df))
+    for i, row in sheet.df.iterrows():
+        if problem := _check_missing_translations_one_row(int(str(i)), row, all_cols):
+            problems.append(problem)
     if problems:
         return MissingTranslationsSheetProblem(sheet.excel_name, sheet.sheet_name, problems)
     return None
 
 
-def _check_for_missing_translations_one_column_level(
-    columns: list[str], df: pd.DataFrame
-) -> list[MissingNodeTranslationProblem]:
-    # column level refers to the hierarchical level of the nodes. eg. ["en_1", "de_1", "fr_1", "it_1", "rm_1"]
-    problems = []
-    for i, row in df.iterrows():
-        if problem := _check_for_missing_translations_one_node(row, columns, int(str(i))):
-            problems.append(problem)
-    return problems
+def _check_missing_translations_one_row(
```

Comment:
I changed it to row level because as a user it is nicest to have all the missing translations sorted by row and one row should not be there more than once. The comments will also be easy to integrate this way.